### DISCUSSION TITLE
Which default fons are supported in Mantine?

### DISCUSSION THREAD

**Discussion by UltrumDev** [mantinedev/mantine#4640] [Q&A] [answered]
In _MantineProvider_ **theme** prop you can set Mantine to use custom font-family. Where do fonts come from and is there a list of supported fonts? Asking because some fonts like Comfortaa are ignored.

```
<MantineProvider
        withGlobalStyles
        withNormalizeCSS
        theme={{
          colorScheme: "dark",
          fontFamily: "Comfortaa, cursive",
        }}
      >
```

Output: just cursive font, not Comfortaa

**Comment by rtivital** [comment, marked as answer]
You need to load these fonts on your side. Usually, it is done with CSS or `<link />` tag
